search/http.go

```diff
@@ -70,8 +79,10 @@ func (e *Engine) RunHttp() error {
 	router := httprouter.New()
 	router.GET("/find/:query", makeRoute(httpFind))
 	router.PUT("/config/:database", makeRoute(httpConfigSet))
+	router.PUT("/rename_table/:external/:internal", makeRoute(httpConfigRenameTable))
```

Comment:
Change these APIs to `/config/rename_table` and `/config/delete_table`. Obviously the C++ code will need to change too.